src/pytometry/tl/clustering/_flowsom.py

```diff
@@ -10,6 +11,9 @@
 from sklearn.cluster import AgglomerativeClustering
 from tqdm.auto import tqdm
 
+SeedLike = int | np.integer | Sequence[int] | np.random.SeedSequence
```

Comment:
Nit:  These could also live in a types.py or something on the root level as they can be shared across a few of Pytometry's modules.